styx-common/src/test/java/com/spotify/styx/util/TimeUtilTest.java

```diff
@@ -187,53 +191,149 @@ public void shouldAddOffsetWithWeek() {
 
   @Test
   public void cronScheduleShouldNotEqualEquivalentWellKnownSchedule() {
-    assertFalse(Schedule.parse("0 * * * *").equals(Schedule.HOURS));
+    assertNotEquals(Schedule.parse("0 * * * *"), Schedule.HOURS);
   }
 
   @Test
   public void shouldGetCorrectNumberOfInstants() {
-    final Instant firstTimeHours = Instant.parse("2016-01-19T00:00:00.00Z");
-    final Instant lastTimeHours = Instant.parse("2016-01-19T09:00:00.00Z");
-    final Instant firstTimeDays = Instant.parse("2016-01-10T00:00:00.00Z");
-    final Instant lastTimeDays = Instant.parse("2016-01-19T00:00:00.00Z");
-    final Instant firstTimeWeeks = Instant.parse("2016-01-11T00:00:00.00Z");
-    final Instant lastTimeWeeks = Instant.parse("2016-01-18T00:00:00.00Z");
-    final Instant firstTimeMonths = Instant.parse("2010-01-01T00:00:00.00Z");
-    final Instant lastTimeMonths = Instant.parse("2016-01-01T00:00:00.00Z");
-
-    assertThat(instancesInRange(firstTimeHours, lastTimeHours, Schedule.HOURS), is(9));
-    assertThat(instancesInRange(firstTimeDays, lastTimeDays, Schedule.DAYS), is(9));
-    assertThat(instancesInRange(firstTimeWeeks, lastTimeWeeks, Schedule.WEEKS), is(1));
-    assertThat(instancesInRange(firstTimeMonths, lastTimeMonths, Schedule.YEARS), is(6));
+    final Instant firstTimeHours = parse("2016-01-19T00:00:00.00Z");
```

Comment:
Nitpick: `NumberOf` in the names of all these tests should now be removed? E.g. `shouldGetCorrectNumberOfInstants` becomes `shouldGetCorrectInstants`.